src/features/campaigns/layout/AllCampaignsLayout.tsx

```diff
@@ -29,6 +29,10 @@ const AllCampaignsLayout: FunctionComponent<AllCampaignsLayoutProps> = ({
           href: `/calendar`,
           label: messages.layout.calendar(),
         },
+        {
+          href: '/activities',
```

Comment:
Does this new page need an update to breadcrumbs?

Reply:
I'm getting a breadcrumb for "activities" so it looks good to me!